swagger/v1/component_schemas/ECFParticipantAttributes.yml

```diff
@@ -0,0 +1,78 @@
+description: "The data attributes associated with an ECF participant"
```

Comment:
Should we also delete the old version of this file and all the other incorrectly named yml files?

Reply:
Yes. Weirdly enough they weren't on my local machine. Could be a Mac case-indifferent filing system thing.